pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -169,6 +170,9 @@ def stopping_condition(op: Operator) -> bool:
     if isinstance(op, qml.ControlledQubitUnitary):
         return True
 
+    if isinstance(op, qml.MPSPrep):
```

Comment:
Is `StatePrep` missed here?

Reply:
I don't think so because the test for `StatePrep` past :thinking: